go.mod

```diff
@@ -12,21 +12,24 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spinkube/spin-operator v0.1.0
 	github.com/stretchr/testify v1.9.0
-	k8s.io/api v0.29.2
-	k8s.io/apimachinery v0.29.2
+	helm.sh/helm/v3 v3.14.3
+	k8s.io/api v0.29.3
+	k8s.io/apimachinery v0.29.3
```

Comment:
Why is this included if it isn't a direct dependency of the plugin?

Reply:
what do you mean by "this" `k8s.io/apimachinery`? As part of the PR I added `helm` as a dependency maybe I've to `go mod tidy` it again?